HQSmokeTests/testCases/test_01_menu_visibility.py

```diff
@@ -1,9 +1,9 @@
-from HQSmokeTests.Pages.homePage import HomePage
-from HQSmokeTests.TestBase.environmentSetupPage import EnvironmentSetup
+from HQSmokeTests.testPages.homePage import HomePage
+from HQSmokeTests.testCases.BaseTest import BaseTest
 from HQSmokeTests.UserInputs.userInputsData import UserInputsData
 
 
-class MenuVisibilityTests(EnvironmentSetup):
+class TestMenuVisibility(BaseTest):
 
     def test_01_reports_menu_visibility(self):
         driver = self.driver
```

Comment:
where does `self.driver` come from? I think this was from the previous base class. Now that `driver` is a fixture you can have it injected into the test method:
```python
def test_01_reports_menu_visibility(self, driver):
    ....
```

This also means that having test classes is not really necessary and you could just have plain functions in the module.